docker-compose.yml

```diff
@@ -22,34 +22,22 @@ services:
 
   db:
     image: mysql:latest
-    ports:
-      - "3306:3306"
     environment:
       MYSQL_DATABASE: moneyapp
       MYSQL_USER: user
       MYSQL_PASSWORD: password
       MYSQL_RANDOM_ROOT_PASSWORD: '1'
 
   otel-collector:
-    image: otel/opentelemetry-collector:latest
-    ports:
-      - "4317:4317"
+    image: otel/opentelemetry-collector:0.86.0
     volumes:
-      - "./otel-collector-config.yaml:/etc/otel-collector-config.yaml"
+      - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml:ro
```

Comment:
was this necessary?

Reply:
yes intentional and I would prefer it be there :)